marathon-gradle-plugin/src/main/kotlin/com/malinskiy/marathon/MarathonPlugin.kt

```diff
@@ -116,6 +114,7 @@ class MarathonPlugin : Plugin<Project> {
                         testVariant = variant
                     )
                     (this as MarathonScheduleTestsToWorkerTask).componentInfo = componentInfo
+                    finalizedBy(marathonWorkerTask)
```

Comment:
If `marathonWorkerTask` is passed as null to this function, the code will raise NPE afaik.

Reply:
I assume that `marathonWorkerTask` is never null because it's initialized under the same condition `if (properties.isCommonWorkerEnabled)`.